### PR TITLE
Add tooltip close button and enable opening tooltips when a draw tool is active

### PR DESCRIPTION
**Pull request by kellyi**
## Overview

This PR makes two adjustments to the draw window tooltips: it adds a close button in the top right corner, and also enables opening the tooltips when a draw tool is active.

Since the clicking anywhere still closes the popovers, the x to close button works primarily as a highlightable thing to click to close it. It's handled here by adding a string template to the popover including the X to close button which then gets rendered as the markup for the popover div.

Opening the tooltips when a draw tool's active took a big to figure out, but it looks like the tooltips were activated when `onShow` was called for free draw or RWD, but then got deactivated if the `render` method was called again (which happens when the component gets new props from the model). To fix this, I abstracted out an `activatePopovers` function for both free draw and RWD, which is called in `onShow` and then again in `onRender`.

Connects #1793 

### Demo

mmw-tooltip-interactivity

### Notes

Still investigating this part to see if it's common/desirable behavior:

> when clicking anywhere to close the tooltip, you may click on another interactive element of the app. It's not clear if this is desirable.

## Testing Instructions

 * get this branch, then rebundle and view it in the browser
 * click on a draw tool, then open a popover and verify that you see the X button in the top right corner, that the cursor turns into a pointer above the button, and that clicking it closes the popover
 * select free draw, sq km, and the rwd options in series and verify that when each of them is selected the popovers still work as usual in that draw pane slideout

### CODE REVIEW

File: src/mmw/js/src/draw/templates/delineationOptions.html [hunk 1]
```diff
@@ -16,33 +16,33 @@
               {{ 'disabled' if toolsDisabled }}>
```
Comment (caseycesari) [inline src/mmw/js/src/draw/templates/delineationOptions.html]: Consider adding a comment to this commit to note why this change was made.

**Comment by kellyi** [comment]
> They do not work in Firefox or Safari

Oof, I think this is because I changed them from `a` elements to `buttons`. Will fix.

**Comment by kellyi** [comment]
Pushed a new set of commits without the `a` -> `button` change, which was what broke things in Safari & Firefox.

**Comment by kellyi** [comment]
> when clicking anywhere to close the tooltip, you may click on another interactive element of the app. It's not clear if this is desirable.

Investigating this a bit and if it makes a difference, the cursor and the interface generally do change when the cursor's placed over an interactive element even when the tooltip is open, which I suppose indicates that the element is still interactive even if the tooltip's open.

The least apparent version of this is the map, though, which also takes up the most space. The cursor changes in a way that indicates it's still interactive, but there's no "element highlighting" like happens with the zoom controls or the layer selector or a link when a tooltip's open.

A single click doesn't have any effect, but a double click when the tooltip's open will close the tooltip and zoom the map, and a click-and-hold will dismiss the tooltip and make the map draggable.

**Comment by kellyi** [comment]
Thanks! I'm going to merge this and I think I'll make another card specifically for deciding whether or not we should disable user interactions when a popover is open.
